### PR TITLE
Upgrade flow to v0.27.1

### PR DESCRIPTION
**Pull request by pattyshack**
Closes #???

## Description

Upgrade flow to v0.27.1

______

For contributor use:

- [x] Targeted PR against `master` branch
- [x] Linked to GitHub issue with discussion and accepted design OR link to spec that describes this work
- [x] Code follows the [standards mentioned here](https://github.com/onflow/flow-emulator/blob/master/CONTRIBUTING.md#styleguides)
- [x] Updated relevant documentation
- [x] Re-reviewed `Files changed` in the GitHub PR explorer
- [x] Added appropriate labels

### CODE REVIEW

File: server/backend/backend.go [hunk 1]
```diff
@@ -28,7 +28,7 @@ import (
 	jsoncdc "github.com/onflow/cadence/encoding/json"
 	sdk "github.com/onflow/flow-go-sdk"
 	"github.com/onflow/flow-go/access"
-	fvmerrors "github.com/onflow/flow-go/fvm/errors"
+	fvmerrors "github.com/onflow/flow-go/fvm/errors" // nolint:typecheck
```
Comment (sideninja) [inline server/backend/backend.go:31]: why do we need this ignored?

Comment (pattyshack) [reply server/backend/backend.go:31]: the linter is incorrectly reporting this as an error (i didn't want to spent too much time looking into this)

Comment (sideninja) [reply server/backend/backend.go]: if the linter has these bugs shouldn't we rather just use an older version then?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -39,11 +39,14 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '1.18.x'
       - name: Run golangci-lint
-        uses: golangci/golangci-lint-action@v2.3.0
+        uses: golangci/golangci-lint-action@v3
```
Comment (sideninja) [inline .github/workflows/ci.yml:46]: we should make sure the makefile is installing the same golangci linter

File: server/backend/backend.go (hunk 1 above)
Comment (pattyshack) [reply server/backend/backend.go]: looks like the linter was compiled using the wrong go version.  these are fixed now

File: .github/workflows/ci.yml (hunk 2 above)
Comment (pattyshack) [reply .github/workflows/ci.yml:46]: bumped version in make file

**Review by sideninja — APPROVED** [review]
Thank you. LGTM

**Pull request merged**